src/common/tests/checksum_timing.c

```diff
@@ -341,19 +450,23 @@ main(int argc, char *argv[])
 #ifdef MCHECKSUM_SUPPORT
```

Comment:
do we still need mchecksum?

Reply:
It's interesting to me. This utility is a good way to compare different checksum libraries and implementations. For completion sake I left mchecksum in, but it's not really needed. I'll add a ticket to remove it in the future if you feel it adds too much noise.